### PR TITLE
fix(cli): exit with a status code of 1 if script fails

### PR DESCRIPTION
**Pull request by davezuko**

### CODE REVIEW

File: flow/check.js [hunk 1]
```diff
@@ -1,9 +1,4 @@
 const cp = require('child_process')
 const flow = require('flow-bin')
 
-try {
```
Comment (davezuko) [inline flow/check.js]: Remove attempted workaround.

File: cli.js [hunk 2]
```diff
@@ -20,7 +20,11 @@ const executeScript = (script, args) => {
   const ext = path.extname(script)
   const program = extProgramMap[ext]
   if (!program) throw new Error(`ta-script has no program assigned to handle the "${ext}" ext`)
-  cp.execSync(`${program} ${script} ${args}`, {stdio: 'inherit'})
+  try {
+    cp.execSync(`${program} ${script} ${args}`, {stdio: 'inherit'})
+  } catch (e) {
+    process.exit(1)
```
Comment (davezuko) [inline cli.js]: Error will already be logged via `stdio`, but we just need to make sure to exit properly.

**Comment by davezuko** [comment]
Tested this locally and seems to do the trick.

File: cli.js (hunk 2 above)
Comment (levithomason) [inline cli.js]: This is still exiting with 0:

image

File: flow/check.js (hunk 1 above)
Comment (levithomason) [inline flow/check.js]: guess is that exit codes are not making it out of the cli all together, see above.  will need to try some other way of getting exit 1 bubbled.

File: cli.js (hunk 2 above)
Comment (davezuko) [inline cli.js]: Really? Will have to re-test but it worked for me.

**Comment by levithomason** [comment]
:accept: other bug will be fixed in another PR

**Pull request merged**